devel/asio/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                  1.0
+PortGroup                   github 1.0
+
+github.setup                chriskohlhoff asio 1.12.0 asio-
+git.branch                  [strsed [join ${github.tag_prefix}]${github.version}[join ${github.tag_suffix}] {g/\./-/}]
+license                     Boost-1
+description                 Asio C++ Library.
+long_description            Asio is a cross-platform C++ library for network and low-level I/O programming that provides developers with a \
+                            consistent asynchronous model using a modern C++ approach.
+homepage                    http://www.think-async.com
+maintainers                 {gmail.com:g.litenstein @Lord-Kamina} openmaintainer
+categories                  devel
+platforms                   darwin
+
+checksums                   rmd160  08cb85fcc1d7c26fa2805d1b6a0f718d8fefb090 \
+                            sha256  657bf83c883d429259d02a5efe010b63cbb571dc344fd13560151fc2a6b5292c \
+                            size    1195583
+
+use_autoconf                yes
+autoconf.dir                ${worksrcpath}/${github.project}
+configure.dir               ${worksrcpath}/${github.project}
+autoconf.cmd                ./autogen.sh
+
+depends_build               port:autoconf port:automake
+depends_lib                 port:boost
+
+configure.cmd               ./configure
+configure.cxxflags-append   -std=gnu++14
+configure.args              --prefix=${prefix}
```

Comment:
Could also be a left-over from when I had everything in a single file but if I added that it was probably because at some point during testing it polluted /usr/local